core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -2001,7 +2001,8 @@ public String toString() {
               + "storage layer runs out of space. Valid options include "
               + "`alluxio.worker.block.evictor.LRFUEvictor`, "
               + "`alluxio.worker.block.evictor.GreedyEvictor`, "
-              + "`alluxio.worker.block.evictor.LRUEvictor`.")
+              + "`alluxio.worker.block.evictor.LRUEvictor`, "
+              + "`alluxio.worker.block.evictor.PartialLRUEvictor`.")
```

Comment:
why add this?

Reply:
@madanadit I found `alluxio.worker.block.evictor.PartialLRUEvictor` is not included in the description in PropertyKey.java for this key so I added this to the description part.